client/lib/analytics/index.js

```diff
@@ -197,7 +197,7 @@ const analytics = {
 	initialize: function( user, superProps ) {
 		analytics.setUser( user );
 		analytics.setSuperProps( superProps );
-		analytics.identifyUser();
+		analytics.identifyUser( user.get().username, user.get().ID );
```

Comment:
We should probably assign the result of `user.get()` to a `const` & reference that instead of calling twice.

Reply:
Something like:

```suggestion
		const userData = user.get();
		analytics.identifyUser( userData.username, userData.ID );
```